### PR TITLE
Handle login errors

### PR DESCRIPTION
**Pull request by 252afh**
- contact email as setting
- login failure template path as setting
- replace all throws with render of error template
- updated test project with template and settings

### CODE REVIEW

File: automatilib/cola/views.py [hunk 1]
```diff
@@ -28,11 +30,19 @@
...
     "LOGIN_REDIRECT_URL",
     "LOGIN_URL",
+    "LOGIN_FAILURE_TEMPLATE_PATH",
+    "CONTACT_EMAIL",
 )
 for setting_key in SETTINGS:
     if not getattr(settings, setting_key):
         raise ImproperlyConfigured(f"{setting_key} must be set")
 
+try:
```
Comment (gecBurton) [inline automatilib/cola/views.py:40]: could/should this be a one-time operation that raises a `ImproperlyConfigured` error if it cant find the template?

File: automatilib/cola/views.py [hunk 2]
```diff
@@ -111,13 +121,25 @@ def get(
         self.pre_login()
 
         if not (cola_cookie := request.COOKIES.get(settings.COLA_COOKIE_NAME, None)):
+            error_response = render(
```
Comment (gecBurton) [inline automatilib/cola/views.py]: this code appears 3 times, could be a function?

File: example_project/templates/auth/login-error.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (gecBurton) [inline example_project/templates/auth/login-error.html:1]: should we use the gov styling?

**Review by gecBurton — APPROVED** [review]
3 non blocking comments added, otherwise LGTM

File: example_project/templates/auth/login-error.html (hunk 3 above)
Comment (252afh) [reply example_project/templates/auth/login-error.html:1]: This is just a stand-in template so the settings will load. Each app will provide the template to use

**Pull request merged**